src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+	
+	private Person personInfo;
+	private Tag tagInfo;
+	private String taggingDetail;
```

Comment:
Using string is a risky choice to keep track of the addition/deletion as it is very error-prone .
 You can consider the alternative of defining a enumeration for OperationType (e.g.) with "ADD" and "DELETE" as values.